tests/e2e/package.json

```diff
@@ -1,8 +1,11 @@
 {
   "name": "e2e",
   "version": "1.0.0",
+  "scripts": {
+    "test": "npm i && dojo upgrade app"
+  },
   "dependencies": {
-    "@dojo/cli-upgrade-app": "file:../../dist/dojo-cli-upgrade-app-3.0.2.tgz",
+    "@dojo/cli-upgrade-app": "file:../../dist/release",
```

Comment:
This is better than pointing to a stale tarball. Hope to use this as part of ci tests if we get a `-y` flag @see #40